packages/template-retail-react-app/translations/de-DE.json

```diff
@@ -1260,7 +1260,7 @@
     "defaultMessage": "Bitte geben Sie Ihre Telefonnummer ein."
   },
   "use_address_fields.error.please_enter_your_postal_or_zip": {
-    "defaultMessage": "Bitte geben Sie Ihre {postalOrZip} ein."
```

Comment:
Later, the translation team will provide us with the complete translation of this.